cpp/ps0005/150daysprogramming.cpp

```diff
@@ -31,130 +16,64 @@ int main()
     char op;
     cin >> op;
     cout << endl << endl;
-    bool isSt1Float = isFloat(st1);
-    bool isSt2Float = isFloat(st2);
-
-
-
-
-
     float num1, num2;
     num1 = stof(st1);
     num2 = stof(st2);

```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input